generator/nidaqmx_python_generator/metadata/doc/functions.md

```diff
@@ -0,0 +1,194 @@
+# ``DAQmx`` function metadata format
+
+## Example
+
+```python
+'CreateAIAccel4WireDCVoltageChan': {
+        'c_function_name' : 'CreateAIAccel4WireDCVoltageChan', 
+        'calling_convention':'StdCall',
+        'description': ' Creates channel(s) that use a Wheatstone bridge to measure force or load. Use this instance with sensors whose...',
+        'isFactory': False,
+        'python_class_name':'AIChannelCollection',
+        'handle_parameter':
+        {
+            "accessor": "self._handle", 
+            "ctypes_data_type": "lib_importer.task_handle", 
+            "cviName": "taskHandle"
+        },
+        'adaptor_parameter': {
+            "adaptor": "self._create_chan(counter, name_to_assign_to_channel)", 
+            "data_type": "nidaqmx._task_modules.channels.co_channel.COChannel", 
+            "direction": "output", 
+            "description": "Indicates the newly created channel object."
+         }, 
+        'parameters': [
+            {
+                ...
+            }
+        ],
+        'returns': 'int32',
+    }
+```
+
+## key for functions
+
+-`'function name'`
+    - The name of the function.
+    - This the key for the function
+    - This is used when generating the function definition.
+
+- `'c_function_name'`
+    - The name of the c function to be called when using the function.
+    - During code generation, this is used to during the definition and the call of the c function.
+
+- `'description'`
+    - The description of the function.
+    - This is used to define the doc string of the function.
+
+- `'calling_convention'`
+    - The calling convention to be followed when using the c functions.
+    - Possible values
+        - 'StdCall'
+        - 'Cdecl'
+- `'handle_parameter'`(Optional)
```

Comment:
When writing English, always leave a space between parentheses and the surrounding words.